src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      class="form"
+      data-qa="big">
+
+      <label for="search1" class="search-box">
+        <img src="images/Search.svg" class="icon"
+        alt="search"></img>
```

Comment:
```suggestion
        <img
          src="images/Search.svg"
          class="icon"
          alt="search"
        >
```
pay attention to the code style